src/client/power-pages/create/CreateTypes.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
interfaces should start with I like ITemplate.

Reply:
Sure, will update all interfaces.